scripts/tests/create_test_clusters.sh

```diff
@@ -3,9 +3,15 @@
 # Change to the directory where the script is located
 cd "$(dirname "$0")"
 
+./../version.sh
```

Comment:
IMO this should be optional, may be add a new argument to the script to have the option to validate the version or not.

Reply:
Since version incompatibility issues are quite common and not always the most intuitive to detect, I'd rather have this as the implied way and instead introduce a flag `--skip-version-check` to skip the version check.